router/core/factoryresolver.go

```diff
@@ -4,12 +4,11 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/wundergraph/graphql-go-tools/v2/pkg/engine/argument_templates"
 	"net/http"
 	"net/url"
 	"slices"
 
-	"github.com/wundergraph/graphql-go-tools/v2/pkg/engine/argument_templates"
-
 	"github.com/buger/jsonparser"
```

Comment:
I know we don't have import sorting as a linter, but can this please go with the other graphql-go-tools/v2 imports 🥺

Reply:
I see your point but it is not very productive to enforce a rule that we didn't agree on. If we want to enforce it we should align on it as a team and automate it. Once option is to use `goimports` which formats imports and does go fmt afterwards. There is IDE support for Goland and VsCode.